test/node/features/restify.feature

```diff
@@ -1,4 +1,4 @@
-@skip_node_18
+@skip_node_12
```

Comment:
new restify version doesn't work on 12 because of a syntax error:

```
/Users/dan.skinner/bugsnag-js/test/node/features/fixtures/restify/node_modules/sonic-boom/index.js:393
    cb?.()
```